subcommand/server-acl-init/command_test.go

```diff
@@ -1663,11 +1663,11 @@ func TestRun_ACLReplicationTokenValid(t *testing.T) {
 }
 
 // Test that if acl replication is enabled, we don't create an anonymous token policy.
-func TestRun_AnonPolicy_IgnoredWithReplication(tt *testing.T) {
+func TestRun_AnonPolicy_IgnoredWithReplication(t *testing.T) {
 	// The anonymous policy is configured when one of these flags is set.
 	cases := []string{"-allow-dns", "-create-inject-auth-method"}
 	for _, flag := range cases {
-		tt.Run(flag, func(t *testing.T) {
+		t.Run(flag, func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(flag, func(tt *testing.T) {
```
?

Reply:
Yeah that's how that one looks in master: https://github.com/hashicorp/consul-k8s/blob/master/subcommand/server-acl-init/command_test.go#L1627